package.json

```diff
@@ -101,7 +101,7 @@
   "scripts": {
     "build": "rm -rf lib && babel -D src -d lib && npm run doc",
     "release": "standard-version",
-    "precommit": "lint-staged --verbose",
+    "precommit": "DEBUG=lint-staged* lint-staged",
```

Comment:
It is doesn't work for me. I agree that --verbose option is deprecated but in version 6.

Reply:
Yes, and to me it seems to possibly have happened even earlier. Good thing you updated the dependency now! I'll leave your change to the `precommit` run-script.